src/singleTypePokemon.java

```diff
@@ -0,0 +1,22 @@
+package src;
+
+public class singleTypePokemon extends Pokemon {
+    public singleTypePokemon(String name, Type type, double xp, Integer level, double hp, double weight, double height, Stat attack) {
+        super(name, type, xp, level, hp, weight, height, attack);
+    }
+
+    @Override
+    public void sound() {
+        System.out.println("The Pokemon makes a sound");
+    }
+
+    @Override
+    public void eats() {
+        System.out.println("The Pokemon is eating a berry");
+    }
+
+    @Override
+    public void attacks() {
+        System.out.println("The Pokmeon Attacks");
+    }
```

Comment:
je @override 3x hetzelfde in double, shiny en single type. je kan deze functies dan net zo goed in je Super class definiëren en niet @Override. De bedoeling van een abstracte methode is dat je al je subclasses een bepaalde functionaliteit dwint te hebben, maar dat de implementatie anders KAN zijn.